src/test/java/com/nextbasecrm/tests/MessageTabLinkTest.java

```diff
@@ -0,0 +1,57 @@
+package com.nextbasecrm.tests;
+
+import com.nextbasecrm.base.TestBase;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import utility.ConfigurationReader;
+
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+public class MessageTabLinkTest extends TestBase {
+
+/*
+1. As a user, I should be able to send messages by clicking on Message tab under Active Stream.
+AC-3  User should be able to attach link by clicking on the link icon.
+ */
+
+//  login2.nextbasecrm.com
+
+    @Test
+    public void test() throws InterruptedException {
+        driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
+
+        String username = ConfigurationReader.getProperty("help_desk_username");
+        String password = ConfigurationReader.getProperty("help_desk_password");
+        loginPage.login(username, password);
+
+        messageTab.messageIcon.click();
+        messageTab.linkIcon.click();
+
+
+        Random number = new Random();  // I need random number because text that I write need be different every time
+        int randomNumber = number.nextInt(1000);
+        String textExpected = "click this link number "+randomNumber; // random text and number
+        Thread.sleep(300); // explicit wait doesnt work so I have to use thread sleep
```

Comment:
Try to use waitForPageToLoad from BrowserUtils package

Reply:
I tried waitForPageToLoad now but it doesn't work. Page already loaded, the issue is when I am sendingKeys to first text box, it combines with second sendingKeys. Selenium is sooo fast I need to hold the horse just 0.3 seconds :))

Reply:
Oh, I see. I had same issue. I used explicit wait before I would sendKeys each line wait.until(ExpectedConditions.elementToBeClickable(<Your WebElement>));
<yourWebElement.sendKeys()>
 wait.until(ExpectedConditions.elementToBeClickable(<Your second  WebElement>));
<Your second  WebElement>.sendKeys()

Reply:
Oh, I did not try elementToBeClickable() method before, because I thought this method can only be used in the clickable element. Basically textbox is not clickable but it works anyway! Thanks, today I learn one more thing:))